src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -576,12 +797,15 @@ uct_cuda_copy_md_open(uct_component_t *component, const char *md_name,
         goto err;
     }
 
-    md->super.ops               = &md_ops;
-    md->super.component         = &uct_cuda_copy_component;
-    md->config.alloc_whole_reg  = config->alloc_whole_reg;
-    md->config.max_reg_ratio    = config->max_reg_ratio;
-    md->config.pref_loc         = config->pref_loc;
-    md->config.dmabuf_supported = 0;
+    md->super.ops                = &md_ops;
+    md->super.component          = &uct_cuda_copy_component;
+    md->config.alloc_whole_reg   = config->alloc_whole_reg;
+    md->config.max_reg_ratio     = config->max_reg_ratio;
+    md->config.pref_loc          = config->pref_loc;
+    md->config.enable_fabric     = config->enable_fabric;
+    md->config.dmabuf_supported  = 0;
+    md->sync_memops_set          = 0;
+    md->granularity              = SIZE_MAX;
```

Comment:
pls revert alignment back
```suggestion
    md->super.ops               = &md_ops;
    md->super.component         = &uct_cuda_copy_component;
    md->config.alloc_whole_reg  = config->alloc_whole_reg;
    md->config.max_reg_ratio    = config->max_reg_ratio;
    md->config.pref_loc         = config->pref_loc;
    md->config.enable_fabric    = config->enable_fabric;
    md->config.dmabuf_supported = 0;
    md->sync_memops_set         = 0;
    md->granularity             = SIZE_MAX;
```